systems/identity/identity-application/src/main/kotlin/hs/kr/entrydsm/identity/application/service/AuthService.kt

```
@@ -0,0 +1,165 @@
+            status = AccountStatus.ACTIVE,
+            profile = StudentProfile(
+                name = command.name,
+                phone = command.phone,
+                birthdate = command.birthdate,
+                signupType = command.signupType,
+                updatedAt = now,
+            ),
+        )
+        val savedAccount = try {
+            accountRegistrationPort.register(registration, now)
+        } catch (exception: AccountAlreadyExistsException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)